resources/models/reservation.py

```diff
@@ -490,6 +490,8 @@ def send_reservation_mail(self, notification_type, user=None, attachments=None):
 
         if user:
             email_address = user.email
+        elif email:
```

Comment:
If the email should it be prioritized over the users email?

Reply:
I think user email should be a priority here since all of the users will have associated email address. We only send emails to extra address if the  passed `email` exists.